### PR TITLE
Practice

### PR DESCRIPTION
**Pull request by andrew-budnitskiy**
This's a training issue No1 - application for random int number search.
25.06.2021
Best regards. 
Andrey Budnitskiy

### CODE REVIEW

**Comment by vasilevkin** [comment]
pr comment better in English.

File: Practice1.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,366 @@
+// !$*UTF8*$!
```
Comment (vasilevkin) [inline Practice1.xcodeproj/project.pbxproj:1]: do not commit this or any other files from a standard new app template
rp should contain your changes only.

**Review by vasilevkin — CHANGES_REQUESTED** [review]

File: Practice1.xcodeproj/project.pbxproj (hunk 1 above)
Comment (andrew-budnitskiy) [reply Practice1.xcodeproj/project.pbxproj:1]: ready

**Comment by andrew-budnitskiy** [comment]
conversations

**Pull request closed without merging**